src/linux/backend-v4l2.cpp

```diff
@@ -761,11 +761,12 @@ namespace librealsense
             // next several lines permit to use D457 even if a usb device has already "taken" the video0,1,2 (for example)
             // further development is needed to permit use of several mipi devices
             static int  first_video_index = ind;
-            if (ind == first_video_index || ind == first_video_index + 2 || ind == first_video_index + 4)
+            ind = (ind - first_video_index) % 6; // offset from first mipi video node and assume 6 nodes per mipi camera
```

Comment:
This will not work if a usb device is also connected. It will have 3 video nodes and the logic used here will then not work.